tests/Aspire.Hosting.AWS.Tests/Aspire.Hosting.AWS.Tests.csproj

```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
     <NoWarn>$(NoWarn);CS8002</NoWarn> <!-- AWS CDK packages are not signed -->
+    <AssemblyOriginatorKeyFile>..\..\public.snk</AssemblyOriginatorKeyFile>
```

Comment:
Same comment as above for using `Directory.Build.props`.

Reply:
Done